plugins/bootc.py

```diff
@@ -68,11 +68,55 @@ def set_argparser(parser):
             action="store_true",
             help=_("If pending deployment available, exit 77 (status)"),
         )
+
+        # these options are for 'install'
+        parser.add_argument(
+            "--uninstall",
+            metavar="PKG",
+            action="store",
+            help=_("Remove overlayed additional package (install)"),
```

Comment:
rpm-ostree accepts multiple `--uninstall` args, looks like

```rpm-ostree install foo --uninstall bar --uninstall baz```

so this should be `nargs="+"`

Reply:
Also good catch.